app/views/candidate_interface/referees/review.html.erb

```diff
@@ -6,18 +6,24 @@
 </h1>
 
 <% editable = @application_form.amendable? ? false : true %>
```

Comment:
From IRL convo: we don't need edits to this page because it's only visible when the form is not submitted yet.